workspace.json

```diff
@@ -2862,31 +2862,39 @@
         }
       }
     },
```

Comment:
I added the bare minimum since I am only adding these extensions as they are written. They are served from within Contentful itself but this code is owned by SÍ.

Please let me know if this is inadvisable.